samples/meeting-av-filters/js/README.md

```diff
@@ -0,0 +1,77 @@
+---
+page_type: sample
+description: "This sample illustrates usage of AV filters in Teams meeting."
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+- javascript
+extensions:
+  contentType: samples
+  createdDate: "10/04/2022 09:09:10 AM"
+urlFragment: officedev-microsoft-teams-samples-meeting-av-filters-javascript
+---
+
+# Meeting AV Filters
+
+This sample demos the use of AV filters for Teams meetings.
+
+![filter-app](Images/MeetingFilter.gif)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+2) Install node modules
+
+   Inside node js folder,  navigate to `samples/meeting-av-filters/nodejs/ClientApp` open your local terminal and run the below command to install node modules. You can do the same in Visual Studio code terminal by opening the project in Visual Studio code.
+
+    ```bash
+    npm install
+    ```
+3) We have the run the solution so follow below steps:
+ 
+   A) In a terminal, navigate to `samples/meeting-av-filters/nodejs/ClientApp`
+
+4) Run ngrok - point to port 3978 (pointing to ClientApp)
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+5) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details
+   - `{{GUID}}` with some unique GUID.
+   - `{{VALID DOMAIN}}` with your application's base url, e.g. https://1234.ngrok.io
```

Comment:
valid domain should also include the port number (i.e., same host names with different ports are considered as different domains).

Reply:
Changed it